wazo_sysconfd/plugins/status/http.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from xivo.status import StatusAggregator
+
+router = APIRouter()
+
+
+@router.get('/status', status_code=200)
+def get_status(status: StatusAggregator = Depends(StatusAggregator)):
+    total_status = {'status': 'up'}
```

Comment:
A `status` key is not very helpful. Consider instead a `rest_api` key. The OK value should be `ok` instead, see the reference for `GET /status`: https://wazo-platform.org/contribute/rest

Reply:
Indeed. I fixed it. Thanks!